app/gateway/2.8.x/plan-and-deploy/security/secrets-management/backends-overview.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Supported Vault Backends
+beta: true
+---
+
+{:.warning}
+> **Beta warning:**
+> <br>
+> Secrets Rotation is currently not supported in this version.
+
+The following Vault implementations are supported:
```

Comment:
Perhaps this requires more explanation on how the Vaults behave similarly to plugins and that more can be added to system, and they need to implement interface X.

Reply:
Since this feature is in a beta state and a lot of things can change, I think we shouldn't encourage people _yet_ to build their own plugins atop a potentially breaking API. WDYT?

Reply:
Can you add more information to this overview about supported Vault backends, and an explanation for GET, SET and Rotation Support. This page is very vague, even if in beta state. Thx.